### DISCUSSION TITLE
Change port

### DISCUSSION THREAD

**Discussion by Ramin-R** [vpnhood/VpnHood#511] [Ideas]
Please add a Future to change port on our server
Right now my 443 is busy by v2ray

**Comment by trudyhood** [comment]
You can do this by changing TcpEndPoints.
See this:
https://github.com/vpnhood/VpnHood/wiki/VpnHood-Server-Configuration

or change it via the VpnHood Access Manager Console:
![image](https://github.com/vpnhood/VpnHood/assets/72891124/67a13e82-eb5f-410f-839c-87560b9a362a)
